code/workspaces/infrastructure-api/src/dataaccess/stacksRepository.js

```diff
@@ -11,6 +11,13 @@ function getAll(user) {
     .exec();
 }
 
+function getAllOwned(user) {
```

Comment:
Again fussy but maybe `getAllByUser`?

Reply:
Not quite - `getAll(user)` would perhaps be the natural thing to call `getAllByUser`, and it gets everything the user can see.  `getAllOwned(user)` is only things the user actually owns (rather than has been shared with them).